core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/TransmitterImpl.java

```diff
@@ -184,8 +186,10 @@ public void run() {
         } catch (Throwable t) {
             try {
                 semaphore.release();
-                logger.error("Error in scheduledSend of telemetry items failed. {} items were not sent", telemetriesFetcher.fetch().size());
-                logger.trace("Error in scheduledSend of telemetry items failed. {} items were not sent", telemetriesFetcher.fetch().size(), t);
+                if (!shutdown) {
```

Comment:
should we log errors whenever there is an exception.  i don't understand why it cares about shutdown or not.